apps/dialtone-documentation/docs/components/scrollbar.md

```diff
@@ -0,0 +1,190 @@
+
+<code-well-header>
+  <dt-stack class="d-hmx164 d-w30p d-bar8 d-ba d-bc-default" v-dt-scrollbar>
+    <div v-for="item in items" class="item">
+      {{ item}}
+    </div>
+  </dt-stack>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-default scrollbar" data-overlayscrollbars="host"
+  data-overlayscrollbars-initialize="true">
+  <div class="os-size-observer">
+    <div class="os-size-observer-listener"></div>
+  </div>
+  <div data-overlayscrollbars-viewport="scrollbarHidden overflowXHidden overflowYScroll" tabindex="-1">
+    <div class="item">user 1</div>
+    ...
+  </div>
+  <div class="os-scrollbar os-scrollbar-horizontal os-theme-dark os-scrollbar-auto-hide
+  os-scrollbar-handle-interactive os-scrollbar-cornerless os-scrollbar-unusable">
+    <div class="os-scrollbar-track">
+      <div class="os-scrollbar-handle"></div>
+    </div>
+  </div>
+  <div class="os-scrollbar os-scrollbar-vertical os-theme-dark os-scrollbar-auto-hide
+  os-scrollbar-handle-interactive os-scrollbar-visible os-scrollbar-cornerless">
+    <div class="os-scrollbar-track">
+      <div class="os-scrollbar-handle"></div>
+    </div>
+  </div>
+</div>
+'
+vueCode='
+<div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-default" v-dt-scrollbar>
+  <div v-for="item in items" class="item">
+    {{ item}}
+  </div>
+</div>
+'
+/>
+
+## Usage
+
+Import the directive from dialtone-vue
+
+```javascript
+import { DtScrollbarDirective } from "@dialpad/dialtone-vue";
+```
+
+Install the directive into vue instance
+
+```javascript
+Vue.use(DtScrollbarDirective);
+```
+
+To add a custom overlay scrollbar to a scrollable region, apply the `v-dt-scrollbar` directive to the desired region.
+There is no need to explicitly add an `overflow` property. If the section overflows the available vertical space, a vertical scrollbar will be present. Similarly, if it exceeds the horizontal space, a horizontal scrollbar will appear.
+
+## Characteristics
+
+* Has an overlay style: it appears on top of the content rather than the scrollbar taking up space within the container.
+* It grows when hovering the scrollbar handle for better accessibility.
+* Appears when the mouse enters the scrollable area and disappears on mouse out after a certain time. This can be customized,
+see [variants](#variants).
+* The look and feel will be the same for every browser and OS.
+* Emulates a browser's native scrollbar keyboard and mouse events.
+
+## Variants
+
+To customize the behavior of the scrollbar, you can use different arguments with the directive. The allowed arguments are 'leave' (default), 'never', 'scroll', and 'move'.
+
+### Leave (default option)
+
+Show the scrollbar when the mouse enters the scrollable area. This is the default option, so no argument is needed.
```

Comment:
Given the definition, maybe it should "Enter (default)"

Reply:
Yeah, agree it's weird, will update 👍